modin/pandas/test/test_io.py

```diff
@@ -212,6 +212,44 @@ def eval_to_file(tmp_dir, modin_obj, pandas_obj, fn, extension, **fn_kwargs):
     assert assert_files_eq(unique_filename_modin, unique_filename_pandas)
 
 
+def eval_to_csv_file(tmp_dir, modin_obj, pandas_obj, extension, **kwargs):
+    if extension is None:
+        kwargs["mode"] = "t"
+        kwargs["compression"] = "infer"
+        modin_csv = modin_obj.to_csv(**kwargs)
+        pandas_csv = pandas_obj.to_csv(**kwargs)
+        if modin_csv == pandas_csv:
+            return
+
+        force_read = True
+        modin_file = get_unique_filename(extension="csv", data_dir=tmp_dir)
+        pandas_file = get_unique_filename(extension="csv", data_dir=tmp_dir)
+        with open(modin_file, "w") as file:
+            file.write(modin_csv)
+        with open(pandas_file, "w") as file:
+            file.write(pandas_csv)
+    else:
+        force_read = extension != "csv" or kwargs.get("compression", None)
```

Comment:
how do we compare results for non csv extensions?

Reply:
The file is read with `pandas.read_csv()` and the result is compared to the input `pandas_obj`.